_posts/2023-04-05-ml-deployment-status.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title:  "See machine learning model status in OpenSearch Dashboards"
+authors:
+- naarcha
+date: 2023-04-05
+categories:
+ - technical-post
+meta_keywords: machine learning, ml commons, opensearch dashboards, model-serving framework, deployed models dashboard
+meta_description: Learn how to check your deployed machine learning models deployment status inside OpenSearch Dashboards.
+excerpt: Before OpenSearch 2.6, checking the status of a deployed machine learning (ML) model in OpenSearch could only be done by using the ML Commons Profile API. Although the Profile APi returned data organized by ML nodes, it could not tell you the status of a specific model deployed on that node. 
+---
+
+Before OpenSearch 2.6, checking the status of a deployed machine learning (ML) model in OpenSearch could only be done by using the [ML Commons Profile API](https://opensearch.org/docs/latest/ml-commons-plugin/api/#profile). Although the Profile APi returned data organized by ML nodes, it could not tell you the status of a specific model deployed on that node. 
```

Comment:
@Naarcha-AWS  As @pajuric said could we link to the opensearch 2.6 introduction blog? Here is the article link: https://opensearch.org/blog/introducing-opensearch-2-6/

Reply:
Can do!

Reply:
Done.